.github/workflows/release.yml

```
@@ -1765,11 +1864,15 @@ jobs:
     name: Publish npm packages
     needs: [prepare, publish-release]
-    if: ${{ !inputs.dry_run && !inputs.desktop_validation_only }}
+    if: ${{ github.event_name == 'workflow_dispatch' && !inputs.dry_run && !inputs.desktop_validation_only }}
     runs-on: ubuntu-latest
     permissions:
       id-token: write    # Required for OIDC trusted publishing
       contents: read
+    concurrency:
+      group: release-npm-publication
+      cancel-in-progress: false
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)